### PR TITLE
chore(ci): Openshift PR deploy test

### PR DESCRIPTION
**Pull request by GDamaso**
OPENSHIFT-TEST

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-sustainment-capstone-2024-87-frontend.apps.silver.devops.gov.bc.ca)
- [Backend](https://nr-sustainment-capstone-2024-87-frontend.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-sustainment-capstone-2024/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-sustainment-capstone-2024/actions/workflows/merge.yml)

### CODE REVIEW

**Comment by DerekRoberts** [comment]
Hi @GDamaso!  This is tagged on my board, so let's team up soon.  ;)

**Comment by DerekRoberts** [comment]
@GDamaso @fergmac Close!  Ideas?
```
error: password authentication failed for user "admin"
at Parser.parseErrorMessage (/backend/node_modules/pg-protocol/src/parser.ts:369:69)
at Parser.handlePacket (/backend/node_modules/pg-protocol/src/parser.ts:188:21)
at Parser.parse (/backend/node_modules/pg-protocol/src/parser.ts:103:30)
at Socket.<anonymous> (/backend/node_modules/pg-protocol/src/index.ts:7:48)
at Socket.emit (node:events:519:28)
at Socket.emit (node:domain:488:12)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Socket.Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:191:23) {
length: 101,
severity: 'FATAL',
code: '28P01',
detail: undefined,
hint: undefined,
position: undefined,
internalPosition: undefined,
internalQuery: undefined,
where: undefined,
schema: undefined,
table: undefined,
column: undefined,
dataType: undefined,
constraint: undefined,
file: 'auth.c',
line: '326',
routine: 'auth_failed'
}
[nodemon] app crashed - waiting for file changes before starting...
npm notice
npm notice New minor version of npm available! 10.5.2 -> 10.8.0
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.8.0>
npm notice Run `npm install -g npm@10.8.0` to update!
npm notice
npm ERR! path /backend
npm ERR! command failed
npm ERR! signal SIGTERM
npm ERR! command sh -c nodemon src/server.ts
npm ERR! Log files were not written due to an error writing to the directory: /.npm/_logs
npm ERR! You can rerun the command with `--loglevel=verbose` to see the logs in your terminal
```

File: charts/app/values.yaml [hunk 1]
```diff
@@ -14,8 +14,8 @@ global:
   secrets:
     enabled: true
     databasePassword: ~
```
Comment (kcaparas1630) [inline charts/app/values.yaml:16]: Correct me if I'm wrong, this should be 'admin' too.

Comment (DerekRoberts) [reply charts/app/values.yaml:16]: It's supposed to autogenerate a password if none are provided. Did you not care and want that hard coded?

Comment (kcaparas1630) [reply charts/app/values.yaml:16]: I've only tried commenting 🤣 , I have no clue with how the backend was made 🤪

**Comment by GDamaso** [comment]
> @GDamaso @fergmac Close! Ideas?
> 
> ```
> error: password authentication failed for user "admin"
> at Parser.parseErrorMessage (/backend/node_modules/pg-protocol/src/parser.ts:369:69)
> at Parser.handlePacket (/backend/node_modules/pg-protocol/src/parser.ts:188:21)
> at Parser.parse (/backend/node_modules/pg-protocol/src/parser.ts:103:30)
> at Socket.<anonymous> (/backend/node_modules/pg-protocol/src/index.ts:7:48)
> at Socket.emit (node:events:519:28)
> at Socket.emit (node:domain:488:12)
> at addChunk (node:internal/streams/readable:559:12)
> at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
> at Socket.Readable.push (node:internal/streams/readable:390:5)
> at TCP.onStreamRead (node:internal/stream_base_commons:191:23) {
> length: 101,
> severity: 'FATAL',
> code: '28P01',
> detail: undefined,
> hint: undefined,
> position: undefined,
> internalPosition: undefined,
> internalQuery: undefined,
> where: undefined,
> schema: undefined,
> table: undefined,
> column: undefined,
> dataType: undefined,
> constraint: undefined,
> file: 'auth.c',
> line: '326',
> routine: 'auth_failed'
> }
> [nodemon] app crashed - waiting for file changes before starting...
> npm notice
> npm notice New minor version of npm available! 10.5.2 -> 10.8.0
> npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.8.0>
> npm notice Run `npm install -g npm@10.8.0` to update!
> npm notice
> npm ERR! path /backend
> npm ERR! command failed
> npm ERR! signal SIGTERM
> npm ERR! command sh -c nodemon src/server.ts
> npm ERR! Log files were not written due to an error writing to the directory: /.npm/_logs
> npm ERR! You can rerun the command with `--loglevel=verbose` to see the logs in your terminal
> ```

I found that error yesterday when I started implementing jest. I'm working on solving that right now.

**Comment by GDamaso** [comment]
I didn't know renaming the branch would close the PR. I thought the branch would be reassigned automatically.

I'm merging this branch with main which has the fix for the error we were facing with the Postgres password.

The fix is here:
https://github.com/bcgov/nr-sustainment-capstone-2024/pull/91#event-12935128872
Wiki if you want to know more:
https://github.com/bcgov/nr-sustainment-capstone-2024/wiki/Lessons-Learned

**Review by kcaparas1630 — APPROVED** [review]
🚀

**Review by DerekRoberts — APPROVED** [review]

**Comment by GDamaso** [comment]
Unnecessary after Docker-compose-update:
https://github.com/bcgov/nr-sustainment-capstone-2024/pull/99

**Pull request closed without merging**